<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>307</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboJoystick">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveParams">
       <property name="text">
        <string>Save params</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadParams">
       <property name="text">
        <string>Load params</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="vlAxes"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vlMixers"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
